android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -74,6 +78,10 @@ class PlayerModule(context: ReactApplicationContext) : BitmovinBaseModule(contex
         val playerConfig = playerConfigJson?.toPlayerConfig() ?: PlayerConfig()
         val analyticsConfig = analyticsConfigJson?.toAnalyticsConfig()
         val defaultMetadata = analyticsConfigJson?.getMap("defaultMetadata")?.toAnalyticsDefaultMetadata()
+        val enableMediaSession = playerConfigJson?.getMap("lockScreenControlConfig")
+            ?.toLockScreenControlConfig()?.isEnabled ?: false
+        enableBackgroundPlayback = playerConfigJson?.getMap("playbackConfig")
```

Comment:
I'm not sure about double source of truth (kind of). But basically my understanding is that once media session is created we automatically support background playback as the player instance is held in the media session service and thus won't be killed by the system. Do we really need an extra flag to enable/disable background playback? Has this been discussed at all?

From my point of view playing on the lock screen is already playing in the background (the app isn't visible - in foreground anymore), so I don't really see the value in this extra flag. Just thinking from the Android point of view, I know that this flag exists for iOS already.

What I'm getting at is that in my opinion we might only want to enable backgorund playback where lockscreen controls would be there automatically, but I might not have enough context what the expectations are on iOS or from the customers.

Reply:
Address in fe1e6517f1fe2b63efff420572a4f4f1d439b537 , following the design we decided in the meeting